infra/modules/providers/azure/app-service/README.md

```diff
@@ -0,0 +1,69 @@
+## Azure App Service
+
+Build, deploy, and scale enterprise-grade web, mobile, and API apps running on any platform. Meet rigorous performance, scalability, security and compliance requirements while using a fully managed platform to perform infrastructure maintenance.
+
+More information for Azure App Services can be found [here](https://azure.microsoft.com/en-us/services/app-service/)
+
```

Comment:
General note, please update the usage section of the readme for the service plan module so that cobalt users have more context on how to define and configure there module.

Reply:
Sure, let me open a new PR with the documentation changes for service plan.